slides/01-introduction.qmd

```diff
@@ -351,7 +367,7 @@ ring_test <- testing(ring_split)
 from sklearn import model_selection
 import numpy as np
 np.random.seed(123)
-X, y = abalone.iloc[:,:8], abalone['rings']
+X, y = inspection.drop(columns=['aka_name', 'inspection_date', 'results']), inspection['results']
```

Comment:
I can also drop the name and inspection date from the pinned data, but maybe we can use the inspection date to talk about monitoring?